### DISCUSSION TITLE
Question/request: clarified docs on `bump` in `matchUpdateTypes`

### DISCUSSION THREAD

**Discussion by jamesbraza** [renovatebot/renovate#31491] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Coming from https://github.com/renovatebot/renovate/discussions/15360#discussioncomment-10689648

I am finding myself not quite understanding why one would set `bump` into [`matchUpdateTypes`](https://docs.renovatebot.com/configuration-options/#matchupdatetypes).

I know `bump` comes from [`rangeStrategy`](https://docs.renovatebot.com/configuration-options/#rangestrategy), where it basically allows a version band to grow even if it didn't need to. For example, Renovate can expand `^1.0.0` to `^1.1.0`.

However, I am not understanding why `bump` is an option for `matchUpdateTypes`. For example, compare `matchUpdateTypes": ["minor", "bump"]` with `matchUpdateTypes": ["minor"]`. Why would minor not be eligible for a match, regardless if a version bump is taking place or not?

Thank you in advance for your help

### Logs (if relevant)

n/a

**Comment by rarkins** [comment]
> Why would minor not be eligible for a match, regardless if a version bump is taking place or not?

I don't understand what this question means. Can you rephrase it?

**Reply by jamesbraza** [reply]
Like let's say a package bumps its version from 1.0.0 to 1.1.0, and we want to allow that. Wouldn't `"matchUpdateTypes": ["minor"]` match that?

In other words, I don't understand why `bump` is a possible value for `matchUpdateTypes`, they seem like separate concerns.

---

For even more context, I am wondering why https://github.com/renovatebot/renovate/issues/1342#issuecomment-798823378 has `"bump"` in its `matchUpdateTypes`

**Reply by rarkins** [reply]
> Like let's say a package bumps its version from 1.0.0 to 1.1.0, and we want to allow that. Wouldn't `"matchUpdateTypes": ["minor"]` match that?

Yes

> In other words, I don't understand why `bump` is a possible value for `matchUpdateTypes`, they seem like separate concerns.

It's not important that you understand it to use Renovate, so I suggest you forget all about it

> For even more context, I am wondering why [#1342 (comment)](https://github.com/renovatebot/renovate/issues/1342#issuecomment-798823378) has `"bump"` in its `matchUpdateTypes`

This person appears to have just put every single possible value in

**Reply by jamesbraza** [reply]
Okay I got you and thanks for circling back. It might be nice to expand the docs on less intuitive values like `bump` and `rollback`
